sagemaker-triton/fil_ensemble/on_start.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -e
+
+sudo -u ec2-user -i <<'EOF'
+
+mkdir -p rapids_kernel
+cd rapids_kernel
+wget -q https://rapidsai-data.s3.us-east-2.amazonaws.com/conda-pack/rapidsai/rapids21.06_cuda11.0_py3.8.tar.gz
```

Comment:
We generally try to avoid hard-coding regions - is this file available across AWS regions, or only us-east-2?